.gitignore

```diff
@@ -1,4 +1,5 @@
 *.log
+*.log.*
```

Comment:
Can't the first one be adjusted?

Reply:
It will, but will also catch for example math.logarithms. I would rather be more precise when there is no reason not to.